.github/workflows/datawave-tests.yml

```diff
@@ -55,10 +80,16 @@ jobs:
         server-password: GITHUB_PASSWORD
     - name: Build and Run Unit Tests
       run: |
-        mvn --show-version --batch-mode --errors --no-transfer-progress "-Dstyle.color=always" -Pexamples,assemble,spotbugs -Dmaven.build.cache.enabled=false -Ddeploy -Ddist -T1C clean install -Dsurefire.rerunFailingTestsCount=3 -Dutils -DskipITs -DskipFormat
+          if [[ "${{ needs.configure-runner.outputs.runner }}" == "self-hosted" ]]; then
+            export PATH=/opt/apache-maven-3.9.11/bin:$PATH    
+          fi
```

Comment:
can we get this version of maven into the base so this isn't necessary?

Reply:
Yea, theres a runner update that i think is the right thing to do to ensure that path has the maven version on it. Will work on that and then update this